flytepropeller/pkg/controller/nodes/task/k8s/plugin_manager.go

```diff
@@ -232,6 +320,19 @@ func (e *PluginManager) launchResource(ctx context.Context, tCtx pluginsCore.Tas
 		cfg := nodeTaskConfig.GetConfig()
 		backOffHandler := e.backOffController.GetOrCreateHandler(ctx, key, cfg.BackOffConfig.BaseSecond, cfg.BackOffConfig.MaxDuration.Duration)
 
+		if config.GetK8sPluginConfig().EnableCreatePodGroupForPod {
+			podGroup := createPodGroupForPod(k8sTaskCtxMetadata, pod)
+			err := e.kubeClient.GetClient().Create(ctx, podGroup)
+			if err != nil {
+				if !k8serrors.IsAlreadyExists(err) {
+					reason := k8serrors.ReasonForError(err)
+					return pluginsCore.UnknownTransition, errors.Wrapf(stdErrors.ErrorCode(reason), err, "failed to create volcano podgroup for pod")
+				}
+				// The error is IsAlreadyExists
+				logger.Warnf(ctx, "PodGroup [%s] is already exists. Err: %v", podGroup.Name, err)
+			}
+		}
+
 		err = backOffHandler.Handle(ctx, func() error {
```

Comment:
What about this case?

Reply:
As before, this error will still be handled by [the logic a few lines below](https://github.com/flyteorg/flyte/blob/master/flytepropeller/pkg/controller/nodes/task/k8s/plugin_manager.go#L242).